docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+[//]: # (These are comments and are used for reference and will not show up in the PR)
+# Description
+
+[//]: # (Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change)
+
+Closes # (issue)
+
+# Proof of Concept 
+[//]: # (Attach Image of feature added or fix applied and working)
+
+# Testing
+
+[//]: # (Please describe the tests that you ran to verify your changes. Provide instructions that can be reproduced and provide images of tests passing)
+
+# Checklist:
+[//]: # (Add an x between square brackets to mark item as done)
+- [ ] Follows style guidelines and is properly commented
+- [ ] Code changes generate no new warnings
+- [ ] Explained Testing Strategy
```

Comment:
Again don't need to explain testing strategy, just check off whether or not evidence is provided (screenshots)